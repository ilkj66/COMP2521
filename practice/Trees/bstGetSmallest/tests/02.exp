Enter the preorder traversal of the BST: 
BST:

  6
 / \
3   9
 \
  4

bstreeGetSmallest returned: 3
