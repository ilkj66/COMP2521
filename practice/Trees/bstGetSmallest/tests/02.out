Enter the preorder traversal of the BST: 
BST:

  6
 / \
3   9
 \
  4

bstGetSmallest returned: 3
